docker/docker_files/indexer-config.py.example

```diff
@@ -0,0 +1,28 @@
+from promnesia import Source
```

Comment:
Hmm, not sure about moving this file.. some tests depend on it (and also linked in readme).
But on the other hand it's nice to have some default docker config too.. maybe just create a separate one (could also have less general comments, just essential stuff for running this specific docker setup)?